frontend/packages/dev-console/src/components/import/image-search/ImageSearch.tsx

```diff
@@ -151,8 +142,8 @@ const ImageSearch: React.FC = () => {
         helpText={getHelpText()}
         helpTextInvalid={helpTextInvalid}
         validated={validated}
-        onBlur={() => {
-          handleSearch();
+        onChange={(e: KeyboardEvent) => {
+          handleSearch((e.target as HTMLInputElement).value);
           setFieldTouched('searchTerm', true);
```

Comment:
Pretty sure you don't need to do that now since default `onBlur` on the field will handle this.